jersey-servers/src/main/java/com/palantir/remoting2/servers/jersey/JsonExceptionMapper.java

```diff
@@ -88,6 +90,14 @@ public final Response toResponse(T exception) {
             builder.type(MediaType.TEXT_PLAIN);
             builder.entity(exceptionMessage);
         }
+
+        if (exception instanceof NotAuthorizedException) {
```

Comment:
I could build a separate `NotAuthorizedExceptionMapper` but it wasn't obvious how to do that without repeating all of `#toResponse`...